change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -1164,17 +1166,17 @@ PROCEDURE CLEANUPTEXT
 END [PROCEDURE]
 
 PROCEDURE VALIDATERGDATA
-[* Check the member's new street address to see if it contains
-** a PO Box address per the CU's configuration settings. If a
+[* Check the member's new street and new extra address to see if it
+** contains a PO Box address per the CU's configuration settings.
 ** PO Box address is found, set an error code and create a
```

Comment:
Line 1171 seems to end prematurely - especially when continued to the next line, it reads as: "...set an error code and create a If a note record on the member's account."

Reply:
Good find @tkainz . I've made the change! Can you re-review?